_maps/map_files/KiloStation/KiloStation.dmm

```diff
@@ -5182,23 +5182,23 @@
 /obj/structure/flora/bush/ferny/style_random,
 /obj/effect/decal/cleanable/dirt,
 /turf/open/misc/asteroid/airless,
-/area/space/nearstation)
+/area/station/hallway/secondary/entry)
 "bPP" = (
 /obj/effect/turf_decal/stripes/line,
 /obj/structure/flora/grass/jungle/b/style_random,
 /obj/effect/decal/cleanable/dirt,
 /turf/open/misc/asteroid/airless,
```

Comment:
if yer going to make this area non-space, might be good to also replace the tiles

Reply:
Why? they're still meant to be vented and all

Reply:
yeah this is fine because that's just how kilo's docking port looks (chunnled into the side of a rock).